src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -106,8 +106,38 @@ internal class AssemblyLoader
         {
             try
             {
-                var assembly = Assembly.Load(assemblyName);
-                var foundType = TypeUtils.GetTypes(assembly, type => typeof (T).IsAssignableFrom(type)).First();
+                var exeRoot = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
Lets leave that code as is for now. we had a lot of issues with the right way to load assemblies, and Sergey has been changing that recently too. Lets deal with assembly loading later on, even if it means leaving the type usage as is here for now, ok?